### DISCUSSION TITLE
Allow for Hashnode link

### DISCUSSION THREAD

**Discussion by bgalvao** [jpanther/congo#136] [Ideas]
I am using hashnode over dev.to for blogging. I would like to know if it's an easy addition?

**Comment by jpanther** [comment]
Although this currently isn't bundled in the theme, it's as simple as adding an SVG file into `assets/icons/`. Once you do this you can then use the normal shortcode or links functionality with this icon. Refer to the [icon shortcode documentation](https://jpanther.github.io/congo/docs/shortcodes/#icon) for more details.

**Reply by jpanther** [reply]
As of Congo v2.1.0, the Hashnode icon is bundled with the theme.
